pdf/invoice.tax-tab.tpl

```diff
@@ -24,78 +24,79 @@
-				{if $line.rate == 0}
-					{continue}
-				{/if}
-				{assign var=has_line value=true}
-				<tr>
-					<td class="white">
-						{if !$label_printed}
-							{if $label == 'product_tax'}
-								{l s='Products' d='Shop.Pdf' pdf='true'}
-							{elseif $label == 'shipping_tax'}
-								{l s='Shipping' d='Shop.Pdf' pdf='true'}
-							{elseif $label == 'ecotax_tax'}
-								{l s='Ecotax' d='Shop.Pdf' pdf='true'}
-							{elseif $label == 'wrapping_tax'}
-								{l s='Wrapping' d='Shop.Pdf' pdf='true'}
-							{/if}
-							{assign var=label_printed value=true}
-						{/if}
-					</td>
+        {foreach $bd as $line}
+          {if $line.rate == 0}
+            {continue}
+          {/if}
+          {assign var=has_line value=true}
+          <tr>
+            <td class="white">
+              {if !$label_printed}
+                {if $label == 'product_tax'}
+                  {l s='Products' d='Shop.Pdf' pdf='true'}
+                {elseif $label == 'shipping_tax'}
+                  {l s='Shipping' d='Shop.Pdf' pdf='true'}
+                {elseif $label == 'ecotax_tax'}
+                  {l s='Ecotax' d='Shop.Pdf' pdf='true'}
+                {elseif $label == 'wrapping_tax'}
+                  {l s='Wrapping' d='Shop.Pdf' pdf='true'}
+                {/if}
+                {assign var=label_printed value=true}
+              {/if}
+            </td>
 
-					<td class="center white">
-						{$line.rate} %
-					</td>
+            <td class="center white">
+              {$line.rate} %
+            </td>
 
-					{if $display_tax_bases_in_breakdowns}
-						<td class="right white">
-							{if isset($is_order_slip) && $is_order_slip}- {/if}
-							{displayPrice currency=$order->id_currency price=$line.total_tax_excl}
-						</td>
-					{/if}
+            {if $display_tax_bases_in_breakdowns}
+              <td class="right white">
+                {if isset($is_order_slip) && $is_order_slip}- {/if}
+                {displayPrice currency=$order->id_currency price=$line.total_tax_excl}
+              </td>
+            {/if}
 
-					<td class="right white">
-						{if isset($is_order_slip) && $is_order_slip}- {/if}
-						{displayPrice currency=$order->id_currency price=$line.total_amount}
-					</td>
-				</tr>
-			{/foreach}
-		{/foreach}
+            <td class="right white">
+              {if isset($is_order_slip) && $is_order_slip}- {/if}
+              {displayPrice currency=$order->id_currency price=$line.total_amount}
+            </td>
+          </tr>
+        {/foreach}
+      {/foreach}
 
-		{if !$has_line}
-		<tr>
-			<td class="white center" colspan="{if $display_tax_bases_in_breakdowns}4{else}3{/if}">
-				{l s='No taxes' d='Shop.Pdf' pdf='true'}
-			</td>
-		</tr>
-		{/if}
-
-		</tbody>
-	</table>
+      {if !$has_line}
+      <tr>
+        <td class="white center" colspan="{if $display_tax_bases_in_breakdowns}4{else}3{/if}">
+          {l s='No taxes' d='Shop.Pdf' pdf='true'}
+        </td>
+      </tr>
+      {/if}
```

Comment:
```suggestion
      {if !$has_line}
        <tr>
          <td class="white center" colspan="{if $display_tax_bases_in_breakdowns}4{else}3{/if}">
            {l s='No taxes' d='Shop.Pdf' pdf='true'}
          </td>
        </tr>
      {/if}
```

Reply:
We really need a CI tool to check for TPL indentation 😢 

@PierreRambaud I heard you like CI :trollface: ?

Reply:
Hell yeah, just need to unroll my 250 notifications and I'll check it :D 
Otherwise, use a real IDE :trollface:

Reply:
Nah I'm a junior I use a mouse!

Reply:
Oups sorry :(